### PR TITLE
Main Menu

### PR DESCRIPTION
**Pull request by RobertKeanu**
Implemented the main menu of the game

### CODE REVIEW

File: Assets/Art/Laboratorul_2.pdf
Comment (widdrr) [inline Assets/Art/Laboratorul_2.pdf:1]: Why....

File: Assets/Art/cyberpunk_city_pt_2_by_nikamanoukian_dem93f8-pre.jpg
Comment (widdrr) [inline Assets/Art/cyberpunk_city_pt_2_by_nikamanoukian_dem93f8-pre.jpg:1]: Copyright?

File: Assets/Art/momo-smoking-cyberpunk-city-last-origin-pixel-thumb.jpg
Comment (widdrr) [inline Assets/Art/momo-smoking-cyberpunk-city-last-origin-pixel-thumb.jpg:1]: Copyright?

**Review by widdrr — CHANGES_REQUESTED** [review]

File: Assets/Art/Laboratorul_2.pdf
Comment (RobertKeanu) [reply Assets/Art/Laboratorul_2.pdf:1]: no comment

**Comment by widdrr** [comment]
Finish the Menu (the buttons should work, for now Play should send you to our main scene)

File: Assets/Art/cyberpunk_city_pt_2_by_nikamanoukian_dem93f8-pre.jpg
Comment (RobertKeanu) [reply Assets/Art/cyberpunk_city_pt_2_by_nikamanoukian_dem93f8-pre.jpg:1]: https://www.deviantart.com/nikamanoukian/art/Cyberpunk-City-pt-2-883902356 free for personal use

File: Assets/Art/momo-smoking-cyberpunk-city-last-origin-pixel-thumb.jpg
Comment (RobertKeanu) [reply Assets/Art/momo-smoking-cyberpunk-city-last-origin-pixel-thumb.jpg:1]: same as the other one

**Review by widdrr — APPROVED** [review]

**Review by Petrica81 — COMMENTED** [review]
lgtm

**Review by Petrica81 — APPROVED** [review]

**Pull request merged**